vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManager.java

```diff
@@ -463,6 +469,47 @@ public IChoiceBlock createChoiceBlock(String slideId, List<String> selectedChoic
         return choiceBlockRepo.save((ChoiceBlock) choiceBlock);
     }
 
+    @Override
+    public IBiblioBlock createBiblioBlock(String slideId, String title, String description, Integer contentOrder) {
+        ISlide slide = slideManager.getSlide(slideId);
+        IBiblioBlock bilioBlock = new BiblioBlock();
+        bilioBlock.setDescription(title);
+        bilioBlock.setBiblioTitle(description);
+        bilioBlock.setSlide(slide);
+        bilioBlock.setContentOrder(contentOrder);
+        return biblioBlockRepo.save((BiblioBlock) bilioBlock);
+    }
+
+    @Override
+    public void deleteBiblioBlockById(String id) throws BlockDoesNotExistException {
+        if (id == null) {
+            logger.warn("Attempted to delete biblio block with null id.");
+            return;
+        }
+
+        try {
+            biblioBlockRepo.deleteById(id);
+            logger.info("Successfully deleted biblio block with id: {}", id);
+        } catch (IllegalArgumentException e) {
+            logger.error("Failed to delete biblio block with id: {}. Reason: {}", id, e.getMessage());
```

Comment:
full stracktrace needs to be logged

Reply:
actually never mind, if it's thrown nothing should be logged here

Reply:
removed logger, thinking that is what you wanted